docs/guides/troubleshooting.md

```diff
@@ -102,7 +102,13 @@ You can fix it by following three simple steps:
 
 The terraform apply command should work as expected now.
 
-Alternatively, you can find the hashes of the last 30 provider versions in [`.terraform.lock.hcl`](https://github.com/databrickslabs/terraform-provider-databricks/blob/v0.6.2/scripts/versions-lock.hcl).
+Alternatively, you can find the hashes of the last 30 provider versions in [`.terraform.lock.hcl`](https://github.com/databrickslabs/terraform-provider-databricks/blob/v0.6.2/scripts/versions-lock.hcl). As a temporary measure, you can lock on a prior version by following the following steps:
+
+* Create a `.terraform.lock.hcl` in the root directory of your Terraform project.
```

Comment:
Can you show an example of the content for it?

Reply:
it's liked in https://github.com/databrickslabs/terraform-provider-databricks/blob/v0.6.2/scripts/versions-lock.hcl